QuickStartGuide.md

```diff
@@ -0,0 +1,142 @@
+## Introduction
+
+GitHub Actions can be run in GitHub-hosted cloud or self hosted environments. Self-hosted runners offer more control of hardware, operating system, and software tools than GitHub-hosted runners provide
+
+With just a few steps, you can set up your kubernetes (K8s) cluster to be a self-hosted environment.
+In this guide, we will setup pre-requistes ,deploy Actions Runner controller (ARC) and then target that cluster to run GitHub Action workflows.
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/53718047/181159115-dbf41416-89a7-408c-b575-bb0d059a1a36.png" />
+</p>
+
+
+
+## Setup your K8s cluster
+
+<details><summary><sub>Create a K8s cluster, if not available</sub></summary>
+   <sub>
+If you don't have a K8s cluster, you can install a local environment using minikube - (https://minikube.sigs.k8s.io/docs/start/)
+   </sub>
+</details>
+
+:one: Install cert-manager in your cluster. For more information, see [cert-manager](https://cert-manager.io/docs/installation/)
+
+```shell
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.8.2/cert-manager.yaml
+```
+<sub> *note:- This command uses v1.8.2. Please replace with a later version, if available.</sub>
+
+
+>You may also install cert-manager using Helm. For instructions, see [Installing with Helm](https://cert-manager.io/docs/installation/helm/#installing-with-helm)
+
+
+:two: Next, Generate a Personal Access Token (PAT) for ARC to authenticate with GitHub.
+   - Login to GitHub account and Navigate to https://github.com/settings/tokens/new
+   - Select  **repo**
+   - Click **Generate Token** and then copy the token locally ( we’ll need it later)
+
+
+
+
+## Deploy and Configure ARC
+1️⃣ Deploy  and configure ARC on your K8s cluster. You may use Helm or Kubectl.
+
+
+<details><summary>Helm deployment</summary>
+
+ ##### Create a new file called `custom-values.yaml` containing...
+
+ ```yaml
+authSecret:
+  github_token: REPLACE_YOUR_TOKEN_HERE
```

Comment:
Generally speaking, placing a sensitive value like this one in values.yaml isn't recommended.

Perhaps we'd better use a secret created via `kubectl create secret` like explained in the `Configure Personal Access Token` section? The helm chart uses the secret named `controller-manager` by default as the container for github API credentials so I believe we can just omit this part and move the `Configure Personal Access Token` section before `Install Helm Chart`.

Reply:
Thanks for the feedback. I have now fixed this.